.drone.yml

```diff
@@ -24,7 +24,7 @@ steps:
 - name: audit
   image: node:10
   commands:
-  - yarn run audit
+  - yarn run audit --audit-level=moderate
```

Comment:
Please add to `audit` script in root `package.json` instead (so we get same behavior locally).